reference/sql_workarounds.rst

```diff
@@ -68,7 +68,7 @@ In this technique we use PL/pgSQL to construct and execute one statement based o
       'create temp table results_temp as '
       'select user_id'
       '  from events'
-      ' where user_id = any(array[%s])'
+      ' where user_id = any(array[%s]::bigint[])'
```

Comment:
`user_ids` is `integer[]`, and this line is `bigint[]`, should they match as one of those types or the other?